src/main/java/cz/cvut/kbss/study/security/SecurityConstants.java

```diff
@@ -27,7 +27,32 @@ private SecurityConstants() {
      */
     public static final int SESSION_TIMEOUT = 12 * 60 * 60;
 
-    public static final String ROLE_USER = "ROLE_USER";
+    public static final String user = "user";
```

Comment:
```suggestion
    public static final String user = "ROLE_USER";
```

Reply:
I thought that with this iteration we want to keep same string. So nothing would change when deploying it.

Reply:
done